packages/venia-ui/lib/components/SavedPaymentsPage/creditCard.js

```diff
@@ -1,13 +1,13 @@
 import React, { useCallback, useMemo } from 'react';
+import { shape, string } from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 import { Trash2 as DeleteIcon } from 'react-feather';
 
-import { mergeClasses } from '@magento/venia-ui/lib/classify';
-
+import LinkButton from '../LinkButton';
```

Comment:
Did your auto-sorter do this? Import order isn't worth fixing, but watch your tooling if this wasn't intentional.

Reply:
This was intentional. I have a habit of moving package imports to the top, above the relative imports, and below the npm imports.

Reply:
It was kind of intentional.